backend/apps/volontulo/templates/offers/show_offer.html

```diff
@@ -86,6 +86,19 @@ <h3 class="panel-title">Zaangażowanie czasowe</h3>
 
         </div>
         <div class="col-xs-12 col-sm-6 col-md-4">
+            {% if has_applied %}
+            <div class="panel panel-primary text-center">
```

Comment:
indentation here (BTW we should develop indentation standard for html and I would suggest 2 spaces instead of 4, I know, that this file have 4 spaces and I don't require from You to move to 2 spaces, as in the end we will migrate to Angular, but it's good to be known)